includes/class-newspack-newsletters-ads.php

```diff
@@ -140,8 +95,26 @@ public static function register_ads_cpt() {
 			return;
 		}
 
+		$labels = [
+			'name'               => _x( 'Newsletter Ads', 'post type general name', 'newspack-newsletters' ),
+			'singular_name'      => _x( 'Newsletter Ad', 'post type singular name', 'newspack-newsletters' ),
+			'menu_name'          => _x( 'Newsletter Ads', 'admin menu', 'newspack-newsletters' ),
+			'name_admin_bar'     => _x( 'Newsletter Ad', 'add new on admin bar', 'newspack-newsletters' ),
+			'add_new'            => _x( 'Add New', 'popup', 'newspack-newsletters' ),
+			'add_new_item'       => __( 'Add New Newsletter Ad', 'newspack-newsletters' ),
+			'new_item'           => __( 'New Newsletter Ad', 'newspack-newsletters' ),
+			'edit_item'          => __( 'Edit Newsletter Ad', 'newspack-newsletters' ),
+			'view_item'          => __( 'View Newsletter Ad', 'newspack-newsletters' ),
+			'all_items'          => __( 'All Newsletter Ads', 'newspack-newsletters' ),
+			'search_items'       => __( 'Search Newsletter Ads', 'newspack-newsletters' ),
+			'parent_item_colon'  => __( 'Parent Newsletter Ads:', 'newspack-newsletters' ),
+			'not_found'          => __( 'No Newsletter Ads found.', 'newspack-newsletters' ),
+			'not_found_in_trash' => __( 'No Newsletter Ads found in Trash.', 'newspack-newsletters' ),
+		];
+
 		$cpt_args = [
 			'public'       => false,
+			'labels'       => $labels,
 			'show_ui'      => true,
 			'show_in_menu' => false,
```

Comment:
Any chance we can have the menu item above Settings?

![Screenshot 2020-08-26 at 12 20 52](https://user-images.githubusercontent.com/177929/91297662-9b9ba180-e796-11ea-8522-185b4e811143.png)

Reply:
Done in ab5ac78e350251c0323bc1ee5dee805f4d5dd6aa